docs/_layouts/docs.html

```diff
@@ -4,6 +4,50 @@
 options: full
 ---
 
+
 <div class='round doc clip fill-white keyline-all'>
+
+  <!-- https://github.com/blog/273-github-ribbons -->
+  <a href="https://github.com/mapbox/mapbox-gl-js">
+      <img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png">
+  </a>
+
+  <section class='pad4 keyline-bottom'>
+    <h1 class='space'>Mapbox GL JS</h1>
+    <div class='prose space-bottom2'>
+      <p>Mapbox GL JS is a Javascript &amp; WebGL library that renders interactive maps from <a href='https://www.mapbox.com/blog/vector-tiles'>vector tiles</a> and <a href='https://www.mapbox.com/mapbox-gl-style-spec'>Mapbox GL styles</a><p>
+      <p>It is part of the <a href='https://github.com/mapbox/mapbox-gl'>Mapbox GL ecosystem</a> which includes <a href='https://www.mapbox.com/mapbox-gl-native'>Mapbox GL Native</a>, a compatible renderer written in C++ with bindings for desktop and mobile platforms.</p>
```

Comment:
on the product level, should link to Mapbox Mobile, not mapbox-gl-native (which is also a dead link here)